lib/api/repository/survey_repository.dart

```diff
@@ -1,10 +1,12 @@
 import 'package:injectable/injectable.dart';
 import 'package:lydiaryanfluttersurvey/api/exception/network_exceptions.dart';
 import 'package:lydiaryanfluttersurvey/api/service/survey_service.dart';
+import 'package:lydiaryanfluttersurvey/model/response/survey_detail_response.dart';
 import 'package:lydiaryanfluttersurvey/model/response/surveys_response.dart';
 
 abstract class SurveyRepository {
   Future<SurveysResponse> getSurveys();
+  Future<SurveyDetailResponse> getSurveyDetails(String surveyId);
```

Comment:
should it be singular? Also, can you add a line break between functions for better readability? 😄 
```suggestion
  
  Future<SurveyDetailResponse> getSurveyDetail(String surveyId);
```